### DISCUSSION TITLE
Song Playback - fade out?

### DISCUSSION THREAD

**Discussion by Typhooteaman** [sircharlo/meeting-media-manager#89] [Ideas] [answered]
Thanks again for the really useful media down and conversion to MP4. We are just starting to use the song playback too before meetings, just wondered if it would be possible to fade out the music over a few seconds rather than just abruptly end when the stop button is pressed ready to start the meeting? Does anyone else think that would be helpful? 
Thanks
Rob

**Comment by sircharlo** [comment]
Nice idea, thanks! Slated for next release.

**Reply by sircharlo** [reply]
Implemented in https://github.com/sircharlo/jw-meeting-media-fetcher/commit/021fa28ad6a46338f3c6dacdb0eb622b2f76a158

**Reply by sircharlo** [reply]
Next release planned for end of September

**Comment by Typhooteaman** [comment]
That’s great! Thank you 😊

**Comment by sircharlo** [comment, marked as answer]
See #100

**Reply by e-Feitosa** [reply]
That’s great! Thank you

**Comment by Typhooteaman** [comment]
Thanks for the early release, works well 😀
